.gitignore

```diff
@@ -5,4 +5,336 @@ theme-dev-utils
 vendor/
 *.DS_Store
 *.zip
-*.rej
\ No newline at end of file
+*.rej
+**/*.map
+.ignore
+.svnignore
+.svn
+
+#Ignoring headstart and language files
+*/languages/*
+*/inc/headstart/*
+
+#Retired themes we do not track in git
```

Comment:
I dont think all of these themes are retired. Many of our active themes just haven't been moved to GH yet, see pNEWy-dbH-p2

Reply:
can we actually move them? is there any reason why we shouldn't have them in git?

Reply:
I don't believe a theme has to be 'retired' for us to not track it in git.  There are some themes that are retired that we DO still track in git, and I don't think that's wrong.  This change just ignores any themes that we don't YET track in git.  If we need to make any changes to any of the themes in this list moving forward then we just remove them from the .gitignore and commit them to git.

Either that or, as Maggie suggested, just move them all into git now.  If we don't ignore them or add them though then we're unable to host a git repository in our synced sandbox folders (which, after using that in testing this work, is really darn handy).

Reply:
My preference is to add them all to git as it will make opening issus much easier.

Reply:
👍 I'll plan to bring all absent themes in AFTER this PR so as to keep the waters as clean as possible.